src/System.Management.Automation/engine/remoting/client/JobManager.cs

```diff
@@ -81,51 +81,43 @@ internal void RegisterJobSourceAdapter(Type jobSourceAdapterType)
             Dbg.Assert(jobSourceAdapterType != null, "JobSourceAdapterType should never be called with null value.");
```

Comment:
@KirkMunro, your last commit had 3 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/docs/cmdlet-example/command-line-simple-example.md.https://www.microsoft.com/net/download/core should work
```powershell
retry of URL failed with error: 
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 111
111:                                 throw "retry of URL failed with error: $($_.Message)"

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/docs/cmdlet-example/command-line-simple-example.md.https://www.microsoft.com/net/core#linuxubuntu should work
```powershell
retry of URL failed with error: 
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 111
111:                                 throw "retry of URL failed with error: $($_.Message)"

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/docs/building/windows-core.md.https://www.microsoft.com/net/core#windowscmd should work
```powershell
retry of URL failed with error: 
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 111
111:                                 throw "retry of URL failed with error: $($_.Message)"

```